docs/reference/projectyml.rst

```diff
@@ -54,6 +54,67 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+
+Specify locations from where V2 modules will be installed
+---------------------------------------------------------
+This section explains how to configure your project in order to download v2 modules from any python package repository.
+By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``. There are multiple ways
+to change this behaviour.
+
+Using pip config file at a standard location
+""""""""""""""""""""""""""""""""
```

Comment:
Do you make sure that the length of the title and the length of the title marker match. At the moment we get many ugly warnings regarding this in the compile log.
```suggestion
Using pip config file at a standard location
""""""""""""""""""""""""""""""""""""""""""""""""
```